.github/workflows/deploy-openbridge-static-staging.yaml

```diff
@@ -0,0 +1,16 @@
+name: deploy-open-bridge-static-staging
+
+on:
+  push:
+    branches: ["staging"]
+
+jobs:
+  deploy:
+    permissions:
+      id-token: write
+      contents: read
```

Comment:
isn't this already in the `build-and-deploy.yaml` file?

Reply:
I thought they needed to be in both (and match). I'll check.

Reply:
I tried calling a job with the permissions above from a job that did not have them and got 'The nested job 'build' is requesting 'id-token: write', but is only allowed 'id-token: none'.' . Added the perms to the caller and the job runs. Let's get it in to see if it actually deploys...

Reply:
duplication is definitely not needed.  we have a bunch of repos setup where it's only set once.   for example https://github.com/Sage-Bionetworks-IT/organizations-infra and https://github.com/Sage-Bionetworks-Workflows/nextflow-infra

Reply:
Wonder if it's because I was in a repo that's outside of the orgs... I'll give a try.